Elasticsearch backend test broken

Most of the backend tests using elasticsearch currently fail (unless you have no es server active, in which case they are skipped).

Error message:

```
E           elasticsearch.NotFoundError: NotFoundError(404, 'index_not_found_exception', 'no such index [mock-corpus]', mock-corpus, index_expression)
```

This fixture should be responsible for creating and deleting the mock-corpus index:

https://github.com/UUDigitalHumanitieslab/I-analyzer/blob/f07776dddf249261a3c55ec8a21ebc9967056705/backend/api/conftest.py#L54-L73

Edit: found the issue, this is due to the renaming of the mock corpus index in #756

--- a/backend/api/conftest.py
+++ b/backend/api/conftest.py
@@ -87,7 +87,9 @@ def index_test_corpus(es_client, corpus_name):
     sleep(2)
 
 def clear_test_corpus(es_client, corpus_name):
-    es_client.indices.delete(index = corpus_name)
+    corpus = load_corpus(corpus_name)
+    index = corpus.es_index
+    es_client.indices.delete(index = index)
 
 
 @pytest.fixture

--- a/backend/api/tests/test_termvectors.py
+++ b/backend/api/tests/test_termvectors.py
@@ -46,7 +46,7 @@ def test_find_matches(test_es_client, termvectors_result):
     ]
 
     for query_text, expected_matches in cases:
-        matches = list(termvectors.token_matches(title_tokens, query_text, 'mock-corpus', 'title', test_es_client))
+        matches = list(termvectors.token_matches(title_tokens, query_text, 'ianalyzer-mock-corpus', 'title', test_es_client))
         assert len(matches) == expected_matches
 
 QUERY_ANALYSIS_CASES = [
@@ -96,7 +96,7 @@ def test_query_analysis(test_es_client):
         pytest.skip('No elastic search client')
 
     for case in QUERY_ANALYSIS_CASES:
-        analyzed = termvectors.analyze_query(case['query_text'], 'mock-corpus', 'content.clean', test_es_client)
+        analyzed = termvectors.analyze_query(case['query_text'], 'ianalyzer-mock-corpus', 'content.clean', test_es_client)
         assert sorted(analyzed) == sorted(case['analyzed'])
 
 
@@ -117,7 +117,7 @@ def termvectors_result(test_app, test_es_client):
     id = hit['_id']
 
     termvectors_result = test_es_client.termvectors(
-        index='mock-corpus',
+        index='ianalyzer-mock-corpus',
         id=id,
         term_statistics=True,
         fields = ['title', 'content']

